### PR TITLE
funding: code style cleanup in the funding package

### PR DESCRIPTION
**Pull request by hieblmi**
This PR aims to clean up some code style issues in the `funding` package mostly around line length issues.
Code structure and logic changes are only applied if the linter is asking for it.

### CODE REVIEW

**Review by yyforyongyu — COMMENTED** [review]
Thanks for the PR! This is missing a release note, also curious about how you found these line too long issues?

**Comment by hieblmi** [comment]
> Thanks for the PR! This is missing a release note, also curious about how you found these line too long issues?

Hi @yyforyongyu, will add the release notes and add the change you mentioned.
To spot lines >80 I enable `set colorcolumn=80` in vim which highlights the row 80.

File: funding/manager_test.go [hunk 1]
```diff
@@ -996,7 +1015,8 @@ func assertDatabaseState(t *testing.T, node *testNode,
 		}
 
 		// If we found the channel, check if it had the expected state.
-		if err != channeldb.ErrChannelNotFound && state == expectedState {
+		if !errors.Is(err, channeldb.ErrChannelNotFound) &&
+			state == expectedState {
```
Comment (guggero) [inline funding/manager_test.go:1019]: nit: needs empty line after multi-line `if`.

File: funding/manager_test.go [hunk 2]
```diff
@@ -1061,21 +1084,22 @@ func assertChannelAnnouncements(t *testing.T, alice, bob *testNode,
...
+	// both peers.
 	nodes := []*testNode{alice, bob}
 	for j, node := range nodes {
 		announcements := make([]lnwire.Message, 2)
 		for i := 0; i < len(announcements); i++ {
 			select {
 			case announcements[i] = <-node.announceChan:
 			case <-time.After(time.Second * 5):
-				t.Fatalf("node did not send announcement: %v", i)
+				t.Fatalf("node did not send announcement: %v",
```
Comment (guggero) [inline funding/manager_test.go:1101]: nit: change to "didn't" instead so it can fit?

File: funding/manager_test.go [hunk 3]
```diff
@@ -3495,7 +3567,10 @@ func TestFundingManagerRejectPush(t *testing.T) {
 
 	// Assert Bob responded with an ErrNonZeroPushAmount error.
 	err := assertFundingMsgSent(t, bob.msgChan, "Error").(*lnwire.Error)
-	if !strings.Contains(err.Error(), "non-zero push amounts are disabled") {
+	if !strings.Contains(
```
Comment (guggero) [inline funding/manager_test.go:3570]: nit: Could use `require.ErrorContains` instead.

**Review by guggero — COMMENTED** [review]
Thanks for the fixes!

**Comment by hieblmi** [comment]
> Thanks for the fixes!

Thanks for the review. I've addressed your comments.

**Review by guggero — APPROVED** [review]

**Review by yyforyongyu — APPROVED** [review]
Needs a rebase, otherwise LGTM! Thanks again!

**Pull request merged**